packages/playground/README.txt

```diff
@@ -0,0 +1,57 @@
+=== WordPress Playground ===
+Contributors: antoniosejas, berislavgrgicak, zieladam
```

Comment:
@seanmorris what's your WordPress.org username? I would like to include you as a contributor.

Reply:
Let's start that list with `wordpressdotorg,` similarly to [what the classic-editor plugin does](http://plugins.svn.wordpress.org/classic-editor/trunk/readme.txt).

Reply:
Also let's format this entire file using the wiki syntax with `==` for headers, also similarly to that file referenced above. I'll do the same for block plugin.